src/gopher.cc

```diff
@@ -598,9 +602,13 @@ gopherToHTML(GopherStateData * gopherState, char *inbuf, int len)
 
                     } else if (gtype == GOPHER_INFO) {
                         snprintf(tmpbuf, TEMP_BUF_SIZE, "\t%s\n", html_quote(name));
+                    } else if (gtype == GOPHER_WWW) {
+                        snprintf(tmpbuf, TEMP_BUF_SIZE, "<IMG border=\"0\" SRC=\"%s\"> <A HREF=\"%s\">%s</A>\n",
+                                     icon_url, rfc1738_escape_unescaped(selector), html_quote(name));
                     } else {
                         if (strncmp(selector, "GET /", 5) == 0) {
                             /* WWW link */
+                            icon_url = mimeGetIconURL("internal-link");
```

Comment:
> From https://github.com/squid-cache/squid/pull/889#discussion_r693644055 :
> The internal-link icon fits the bill very well, I have updated to use that icon.
> For consistency I applied it to the GET / hack as well.

Hm... I am worried about this change because, AFAICT, `GET /` selector might be used to help users download something (e.g., a movie). Unlike, `GOPHER_WWW` selectors, the `gtype` corresponding to the `GET /` selector can be one of the other recognized types that have type-specific icons. We should not overwrite that decision unless we did not recognize `gtype` (resulting in an internal-unknown icon earlier).

Moreover, after this PR, Squid will stop applying the `GET /` trick to `GOPHER_WWW` links while still applying it to all other gtypes that produce non-telnet links. Is that a desirable change? Should this `GET /` removal code (continue to) apply to `GOPHER_WWW` links??

Can libwww sources help us address these concerns?

P.S. Originally, I was under the false impression that `GET /` means that gtype is `G` (i.e. the first letter of `GET` is used for gtype) so I did not raise these concerns earlier, but I know see that it is not how the Squid code is actually written -- gtype is always separate from the selector.